src/main/scala/org/kframework/kore/Default.scala

```diff
@@ -0,0 +1,122 @@
+package org.kframework.kore
+
+object implementation {
```

Comment:
I had it as default earlier, but we can't use it since it's a keyword in java. I'm ok with any other word that works